Randolph-dev/Druid_Partnering_Project_React24k
# add new database and changed navbar element

### Related ticket:
No available ticket.

### In this PR:
Added new content for personalized users. Replaced all placholders.
Removed NavBar Finnish translation option.

### Testing:

- Check out to this branch.
- Add new database.
- Send a form to see how the data is being fetched.
